libs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@audius/libs",
-  "version": "0.11.76",
+  "version": "0.11.77",
```

Comment:
When you bump the version, the packge-lock.json should also be bumped for consistency.
I usually run the command `npm version patch` which does it for me.